### PR TITLE
Cisco Catalyst 9200 series switches

### PR DESCRIPTION
**Pull request by gs1571**
I prepared yaml files for:
C9200-24T, C9200-24P, C9200-48T, C9200-48P, C9200L-24T-4G, C9200L-24P-4G, C9200L-48T-4G, C9200L-48P-4G, C9200L-24T-4X, C9200L-24P-4X, C9200L-48T-4X, C9200L-48P-4X

### CODE REVIEW

File: device-types/Cisco/C9200-48P.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
+manufacturer: Cisco
+model: C9200-24T
```
Comment (ymasson) [inline device-types/Cisco/C9200-48P.yaml:2]: C9200-48P, no ?

File: device-types/Cisco/C9200-48T.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
+manufacturer: Cisco
+model: C9200-24T
```
Comment (ymasson) [inline device-types/Cisco/C9200-48T.yaml:2]: C9200-48T

File: device-types/Cisco/C9200L-24T-4G.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+manufacturer: Cisco
+model: C9200-24T-4G
```
Comment (ymasson) [inline device-types/Cisco/C9200L-24T-4G.yaml:2]: C9200L-24T-4G

File: device-types/Cisco/C9200L-24T-4X.yaml [hunk 4]
```diff
@@ -0,0 +1,113 @@
+manufacturer: Cisco
+model: C9200-24T-4X
```
Comment (ymasson) [inline device-types/Cisco/C9200L-24T-4X.yaml:2]: C9200L-24t-4X

File: device-types/Cisco/C9200L-48P-4G.yaml [hunk 5]
```diff
@@ -0,0 +1,185 @@
+manufacturer: Cisco
+model: C9200-48P-4G
```
Comment (ymasson) [inline device-types/Cisco/C9200L-48P-4G.yaml:2]: C9200L-48P-4G

File: device-types/Cisco/C9200L-48P-4X.yaml [hunk 6]
```diff
@@ -0,0 +1,185 @@
+manufacturer: Cisco
+model: C9200-48P-4X
```
Comment (ymasson) [inline device-types/Cisco/C9200L-48P-4X.yaml:2]: C9200L-48P-4X

File: device-types/Cisco/C9200L-48T-4G.yaml [hunk 7]
```diff
@@ -0,0 +1,185 @@
+manufacturer: Cisco
+model: C9200-48T-4G
```
Comment (ymasson) [inline device-types/Cisco/C9200L-48T-4G.yaml:2]: C9200L-48T-4G

File: device-types/Cisco/C9200L-48T-4X.yaml [hunk 8]
```diff
@@ -0,0 +1,185 @@
+manufacturer: Cisco
+model: C9200-48T-4X
```
Comment (ymasson) [inline device-types/Cisco/C9200L-48T-4X.yaml:2]: C9200L-48T-4X

**Review by ymasson — COMMENTED** [review]
I use https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9200-series-switches/nb-06-cat9200-ser-data-sheet-cte-en.html .

For `L` missing or not, I don't know if the error is in the file name or the model

**Comment by gs1571** [comment]
Thanks! All the mistakes were fixed.

**Pull request merged**